VideoLocker/src/main/java/org/edx/mobile/view/MyFriendsCoursesTabFragment.java

```diff
@@ -63,12 +69,17 @@ public void handleCourseClick(EnrolledCoursesResponse model) {
 
     @Override
     public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
+        String platformName = config.getPlatformName();
 
         View view = super.onCreateView(inflater, container, savedInstanceState);
         noFriendsLayout = (LinearLayout) view.findViewById(R.id.friends_course_no_friends_layout);
         TextView shareBtn = (TextView) view.findViewById(R.id.friends_course_no_btn_share_app);
+        CharSequence shareText = ResourceUtil.getFormattedString(getResources(), R.string.btn_share_app, "platform_name", platformName);
+        shareBtn.setText(shareText);
         shareBtn.setOnClickListener(this);
-
+        CharSequence errorText = ResourceUtil.getFormattedString(getResources(), R.string.error_no_friends_connected, "platform_name", platformName);
+        TextView errorMessage = (TextView)view.findViewById(R.id.error_message);
```

Comment:
Again, maybe this variable name could be change to `errorMessageView` to make it clear that this is a widget.